src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -916,8 +900,10 @@ public function load($pFilename)
                                     $relPath = File::realpath(dirname("$dir/$fileWorksheet") . '/' . $relPath);
 
                                     try {
+                                        $secureXml = $this->securityScanner->scan($this->getFromZipArchive($zip, $relPath));
+                                        $cleanedXml = str_replace('<br>', '<br/>', $secureXml);
```

Comment:
This is way too dangerous and cannot be merged as is.

Reply:
You're right, this solves my problem could turn an correct code like "<br></br>" into an unwanted code like "<br/></br>". Thanks for this hint I will prepare another solution.